devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy

```diff
@@ -244,11 +266,11 @@ private Map getTemperatureResult(value) {
 		def v = value as int
 		value = v + offset
 	}
-    def descriptionText
-    if ( temperatureScale == 'C' )
-    	descriptionText = '{{ device.displayName }} was {{ value }}°C'
-    else
-    	descriptionText = '{{ device.displayName }} was {{ value }}°F'
+	def descriptionText
+	if ( temperatureScale == 'C' )
+		descriptionText = '{{ device.displayName }} was {{ value }}°C'
```

Comment:
`Groovy string interpolation syntax is ${device.displayName` or else this shows up as:
f3011b16-656c-4873-8efd-e71169aa1ddc  10:43:16 AM: debug Parse returned [name:water, value:dry, descriptionText:{{ device.displayName }} is dry, translatable:true]

f3011b16-656c-4873-8efd-e71169aa1ddc  10:43:16 AM: debug water
f3011b16-656c-4873-8efd-e71169aa1ddc  10:43:16 AM: debug description: zone status 0x0024 -- extended status 0x00